demisto_sdk/commands/validate/README.md

```diff
@@ -48,8 +48,8 @@ This will validate only changed files from content origin/master branch and will
 compatibility checks.
 <br><br>
 
-`demisto-sdk validate -i -j`
-This will validate all content repo files and including conf.json file and will create the id_set.json file.
+`demisto-sdk validate -j`
+This will validate all content repo files and including conf.json file.
```

Comment:
i think this validates only conf json, not all files

Reply:
No, definitely all files. you can test it.
That's what we talked about yesterday